staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types_test.go

```diff
@@ -45,7 +45,7 @@ func TestVerbsUgorjiMarshalJSON(t *testing.T) {
 	}
 }
 
-func TestVerbsUgorjiUnmarshalJSON(t *testing.T) {
+func TestVerbsUJsonIterUnmarshalJSON(t *testing.T) {
```

Comment:
nit: looks like we have an extra U here :eyes:

Reply:
also in master

Reply:
no, don't fix it here